scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -486,7 +486,7 @@ vep <- readLines(params$vep_file) %>% paste0(collapse="\n") %>% base64_urlencode
 Variant effect prediction (*Ensemble VEP output*):  
 [Download VEP_out.csv](`r sprintf('data:text/csv;base64,%s', vep)`)
 
-```{r csv_output_variant_plot, include = F}
+```{r csv_output_variant_plot, include = F, eval = executeDeconvolution}
```

Comment:
Move this commit up and squash it with "variant_report: adding conditional to deconv-chunks

when there are no sigmuts, Deconvolution can't and should not be performed"